rest-api/api/base_api.py

```diff
@@ -136,11 +136,13 @@ def _make_bundle(self, results, id_field, participant_id):
     if results.pagination_token:
       query_params = request.args.copy()
       query_params['_token'] = results.pagination_token
+
       next_url = main.api.url_for(self.__class__, _external=True, **query_params)
       bundle_dict['link'] = [{"relation": "next", "url": next_url}]
     entries = []
+    inactive_sites = request.args.get('_inactive')
```

Comment:
This argument is specific to the awardee API. It doesn't make sense to look for it here. (And it doesn't make sense to have to add inactive_sites to every to_client_json method.)

What we discussed yesterday was:

1) Change the call to to_client_json on 145 to call _make_resource instead.
2) Override _make_resource in awardee_api to call to_client_json passing in inactive sites.
3) Change HPODao to_client_json to have the optional parameter.

Then you can leave the other DAOs alone, and only have to make a one line change to base_api.